### DISCUSSION TITLE
Partial text disappeared after using doc.subset_fonts()

### DISCUSSION THREAD

**Discussion by susuusus** [pymupdf/PyMuPDF#2942] [Q&A]
### Description of the bug

I am inserting my own ttf file while the size of output pdf is huge. Thereby, I use subset_font() as suggested to reduce the size but partial of my insert text gone. Could you advise, thanks.

### How to reproduce the bug

I am inserting my own ttf file while the size of output pdf is huge. Thereby, I use subset_font() as suggested to reduce the size but partial of my insert text gone. Could you advise, thanks.

### PyMuPDF version

1.23.6 or earlier

### Operating system

Windows

### Python version

3.9

**Comment by JorjMcKie** [comment]
You did not provide all mandatory information to reproduce the issue.
Please complete.

**Comment by JorjMcKie** [comment]
This issue will be closed in 3 more days if you do not provide the missing data.
